src/components/Input.js

```diff
@@ -5,7 +5,6 @@ export default function Input({ ...props }) {
     <input
       {...props}
       className="appearance-none block placeholder-secondary text-secondary bg-primary border rounded p-1 leading-tight focus:outline-none pl-4"
-      data-e2e-id="search-input"
```

Comment:
For future people reading this: we moved `date-e2e-id` from `Input.js` to `Nav.js` since only the `<Input />` component in the `Nav.js` file should have a `data-e2e-id="search-input"`: that id is not something that all `<Input />` components should have.

I'll repeat this message in both the `Nav.js` and `Input.js` files in this review.